build/build-config.yml

```diff
@@ -26,3 +26,8 @@ config:
         image-name: "estimate-service"
 
 # frontend
+  - name: builds/digit-works/frontend/micro-ui/works
+    build:
+      - work-dir: frontend/micro-ui/
+        dockerfile: frontend/micro-ui/web/docker/Dockerfile
+        image-name: works
```

Comment:
Don't give a generic image name like works. Add some suffix so that the frontend project is clearly identifiable.

Reply:
added